### PR TITLE
Versioning

### PR DESCRIPTION
**Pull request by intelmark**
Made changes to create EvApiVersion.h automatically
during make process (similar to HEVC versioning)

Also updated .gitignore to ignore Windows build files

Signed-off-by: Mark Feldman <mark.feldman@intel.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,9 +14,7 @@
...
 *.bat
-Build/VS17/x64
 Bin
-Build/linux/debug/
-Build/linux/release/
-
-
+Build
+!Build/linux/build.sh
+!Build/windows/generate_vs17.bat
```
Comment (1480c1) [inline .gitignore]:
```suggestion
!Build/windows/generate_vs17.bat

```
Please try to add newlines to the eof. That way if someone wants to add something, it doesn't count as a delete and two additions

File: Source/Lib/pkg-config.pc.in [hunk 2]
```diff
@@ -4,6 +4,6 @@ libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 
 Name: SvtVp9Enc
 Description: SVT (Scalable Video Technology) for VP9 encoder library
-Version: @SVT_VP9_VERSION@
+Version: @SVT_HEVC_VERSION_MAJOR@.@SVT_HEVC_VERSION_MINOR@.@SVT_HEVC_VERSION_PATCHLEVEL@
```
Comment (1480c1) [inline Source/Lib/pkg-config.pc.in]:
```suggestion
Version: @SVT_VP9_VERSION_MAJOR @.@SVT_VP9_VERSION_MINOR@.@SVT_VP9_VERSION_PATCHLEVEL@
```

File: .gitignore (hunk 1 above)
Comment (intelmark) [reply .gitignore]: done

File: Source/Lib/pkg-config.pc.in (hunk 2 above)
Comment (intelmark) [reply Source/Lib/pkg-config.pc.in]: I hate copy/paste!
thx

Comment (1480c1) [reply Source/Lib/pkg-config.pc.in]: Same

Comment (1480c1) [reply Source/Lib/pkg-config.pc.in]: Make sure to mark these as resolved

**Comment by tianjunwork** [comment]
Hi @intelmark, I suggest you use **squash and merge** next time if you have many commits in a PR. And edit/combine your commit message at the same time.

**Pull request merged**